src/main/java/cashleh/parser/Parser.java

```diff
@@ -127,6 +127,9 @@ private Expense getExpense(String input) throws CashLehParsingException {
         double expenseAmt;
         try {
             expenseAmt = Double.parseDouble(expenseAmtString);
+            if (expenseAmt < 0) {
+                throw new CashLehParsingException("Please enter a positive expense amount!");
```

Comment:
Maybe can rephrase to "Please enter the absolute amount" because positive expense may sound a bit confusing?